build.sbt

```diff
@@ -147,7 +148,7 @@ goAstGenDlTask := {
   distDir.listFiles().foreach(_.setExecutable(true, false))
 }
 
-Compile / compile := ((Compile / compile) dependsOn goAstGenDlTask).value
+Compile / compile := (Compile / compile).dependsOn(goAstGenDlTask, phpParseDlTask).value
```

Comment:
This doesn't work on Linux, can you please test once and confirm? @pawan-privado

Reply:
I don't have access to a Linux machine ATM but it works fine on macOS. I've DM'd you something on Slack, btw, regarding this.

Reply:
This looked fine to me - I tested on Linux, but did a cleaner version of this just in case

Reply:
> This doesn't work on Linux, can you please test once and confirm? @pawan-privado

Haven't noticed any abnormalities at my end. Can you attach the error log if it still continues to happen? AFAIK, there's no platform specific stage in our build process so it shouldn't fail on Linux systems.